dev/local-environment/README.md

```diff
@@ -13,6 +13,7 @@ The local environment includes:
 - 1 x Ogmios
 - 1 x Kupo
```

Comment:
Maybe we can remove kupo from configuration (with latest changes)?

Reply:
Again, after pc-contracts-cli is totally cleaned up. At this point I would ruin E2E-tests, that depend on pc-contracts-cli.